sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -168,52 +189,36 @@ export interface FormElementCommon {
 }
 
 // @public
-export type FormField = {
+export type FormField = FormUnknownField | FormStringField | FormNumberField | FormDateField | FormTimeField | FormPhoneNumberField | FormIntegerField | FormSelectionMarkField | FormArrayField | FormObjectField | FormGenderField | FormCountryField;
+
+// @public
+export interface FormFieldCommon {
     confidence?: number;
     labelData?: FieldData;
     name?: string;
     valueData?: FieldData;
-} & ({
-    value?: string;
-    valueType?: "string";
-} | {
-    value?: number;
-    valueType?: "number";
-} | {
-    value?: Date;
-    valueType?: "date";
-} | {
-    value?: string;
-    valueType?: "time";
-} | {
-    value?: string;
-    valueType?: "phoneNumber";
-} | {
-    value?: number;
-    valueType?: "integer";
-} | {
-    value?: FormField[];
-    valueType?: "array";
-} | {
-    value?: Record<string, FormField>;
-    valueType?: "object";
-} | {
-    value?: SelectionMarkState;
-    valueType?: "selectionMark";
-} | {
-    value?: string;
-    valueType?: "gender";
-} | {
-    value?: string;
-    valueType?: "country";
-});
+}
 
 // @public
 export interface FormFieldsReport {
     accuracy: number;
     fieldName: string;
 }
 
+// @public
+export interface FormGenderField extends FormFieldCommon {
+    value?: string;
+    // (undocumented)
```

Comment:
what does undocumented mean?

Reply:
It means that this field in this particular interface doesn't have a doc string. I'm trying to figure out if I need to put the same string on all of them or write specific ones that say "this is always X in this variation". The conditions for a user to actually _see_ this particular doc string in Intellisense are rare.